Dockerfile

```diff
@@ -1,35 +1,57 @@
-FROM node:20.2-alpine AS base
-# Installs global dependencies as non root user for security
+FROM node:20-alpine AS base
```

Comment:
Mind grabbing a more detailed version? 

I ran into a hiccup when I tried using the 20-alpine image not long ago. 

That's why I opted for version 20.2. It's possible the issue has been resolved by now, but I'd rather not risk version 20.6 causing any problems again. 

Can't go for LTS since we're taking advantage of some of the newest node features.

Reply:
I chose to upgrade it bc. I did a quick security scan of the image with 20.2 and it reported two high vulnerabilities one being with libssl. However I did not read into the details

Reply:
then lets just pin it to the latest minor if you tested it. (20.5)